server-adapters/resteasy-vertx/src/test/java/org/jboss/resteasy/test/asyncio/actual.txt

```diff
@@ -0,0 +1,45 @@
+0: org.jboss.resteasy.test.cdi.interceptors.resource.InterceptorRequestFilterInterceptor
```

Comment:
Is this actual.txt file used? Same question for the expected.txt

Reply:
OOOPS.

Reply:
force-pushed fixed.